utils/models.py

```
@@ -0,0 +1,178 @@
+
+class PoliticalBiasBert(Model):
+    def __init__(self):
+        super().__init__(
+            AutoTokenizer.from_pretrained("bert-base-cased"),
+            AutoModelForSequenceClassification.from_pretrained(
+                "bucketresearch/politicalBiasBERT"
+            ),
+            True,
+        )
+
+    def predict(self, article_body: str, truncate_tokens: bool) -> Leaning:
```

Ruff

Missing type annotation for `self` in method

(ANN101)